### PR TITLE
feat (accordion): add prop for removing the last accordion item bottom border

### PR DESCRIPTION
**Pull request by patrikhultgren**
## **Hide last border in accordion**  
Add prop for removing the last accordion item bottom border.

## **Issue Linking:**  
https://github.com/scania-digital-design-system/tegel/issues/625

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Run Storybook.
2. Go the Accordion and select the Docs tab.
3. Select "Hide last border".
4. Go to the Canvas tab to see the result.

## **Checklist before submission**
- [x] All existing tests pass

## **Suggested test steps**
- [x] Browser testing (Chrome, Safari, Firefox)
- [x] Added e2e-test

## **Screenshots**  
Before:
before

After:
after

### CODE REVIEW

**Comment by theJohnnyMe** [comment]
@patrikhultgren  - Please, if you can, add the playwright test that will check the functionality of this prop by setting it to true and taking a screenshot of it. 
Great contribution and code looks OK to me. Will be approved after test and confirmation from designers ;)

File: packages/core/src/components/accordion/accordion.tsx
Comment (mJarsater) [inline packages/core/src/components/accordion/accordion.tsx:16]: @theJohnnyMe do we have any other examples of this sort of prop, can't think of any from the top of my head? Just thinking for naming alignment.

File: packages/core/src/components/accordion/accordion.tsx
Comment (mJarsater) [inline packages/core/src/components/accordion/accordion.tsx:23]: Not sure this change is needed? If it adds something we should update the same logic for all components, if not I'd say we skip it.

Comment (patrikhultgren) [reply packages/core/src/components/accordion/accordion.tsx:23]: Otherwise I will get a SonarCloud issue. Is that okay @mJarsater?

Comment (mJarsater) [reply packages/core/src/components/accordion/accordion.tsx:23]: Ah, alright! Is it a code duplication warning?

File: packages/core/src/components/accordion/accordion.tsx
Comment (timrombergjakobsson) [reply packages/core/src/components/accordion/accordion.tsx:16]: Maybe, just go with `hideLastBorder`, as we use hide a lot?

File: packages/core/src/components/accordion/accordion.tsx
Comment (patrikhultgren) [reply packages/core/src/components/accordion/accordion.tsx:23]: I think it was a "no-nested-template-literals" when I was doing something like this:
`tds-accordion ${
          this.modeVariant !== null ? `tds-mode-variant-${this.modeVariant}` : ''
        } ${
          this.noLastBorder ? 'no-last-border' : ''
        }`

File: packages/core/src/components/accordion/accordion.tsx
Comment (patrikhultgren) [reply packages/core/src/components/accordion/accordion.tsx:16]: I have seen a name like `noIcon` in the message component. But of course I can change it if you want to.

Comment (patrikhultgren) [reply packages/core/src/components/accordion/accordion.tsx]: Now I'm using `hideLastBorder`.

**Comment by patrikhultgren** [comment]
> @patrikhultgren - Please, if you can, add the playwright test that will check the functionality of this prop by setting it to true and taking a screenshot of it. Great contribution and code looks OK to me. Will be approved after test and confirmation from designers ;)

Now there is a e2e-test for the removed border.

**Review by nathalielindqvist — APPROVED** [review]
Looks good to me! Thank you for your contribution, great stuff! 🚀

**Review by theJohnnyMe — APPROVED** [review]
Nice, thank you for another great contribution Patrik!

**Comment by theJohnnyMe** [comment]
@patrikhultgren  - Please do update you branch with latest develop changes by "git rebase develop" or by using GitHub UI before "Squash and merge" :)

**Comment by patrikhultgren** [comment]
Thanks! I have updated the branch. I don’t have permission to merge, which is fine.

**Comment by timrombergjakobsson** [comment]
> Thanks! I have updated the branch. I don’t have permission to merge, which is fine.

Ill merge it! Thanks a lot for the contribution!

**Pull request merged**
